### ISSUE TITLE
Pericallsol Lipoma

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7095]
**Preferred term label:**
Parent term will be "Pericallosal Lipoma"

I am submitting sep. issues for the two child terms (curvilinear and tubulonodulal)

**Synonyms**

**Definition (free text, please give PubMed ID)**

Pericallosal lipomas (PCL) are congenital soft masses of adipose cells encapsulated by a thin layer of fibrous tissue, appearing adjacent to the corpus callosum of the brain.
PMID: 30104143

**Parent term (use hpo.jax.org/app)**
I think maybe HP:0006866  Midline central nervous system lipomas

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pai disease ORPHA:1993 is one syndrome that has these lipomas but not the only manifestation as they can occur on their own without any other symptoms. 

**Your nano-attribution (ORCID)**
0000-0002-3302-4610

**Comment by taylordm** [comment]
Maybe this term should be a synonym of this one:

Lipoma of corpus callosum
HP:0006931

though the descriptions don't quite match.

**Comment by pnrobinson** [comment]
Thanks, @taylordm for this request. I see that the term [Lipoma of corpus callosum](https://hpo.jax.org/app/browse/term/HP:0006931) is used to annotate [Frontonasal dysplasia 1](https://omim.org/entry/136760). This annotation seems to derive from this article: [Frontonasal dysostosis in two successive generations](https://pubmed.ncbi.nlm.nih.gov/10564879/). I cannot surrently access that article but am seeing others, e.g. this one https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7975078/ where the lipoma is clearly "peri" (adjacent to) the corpus callosum, and so I think these terms are synonymous. I will try to get the original article and see if this is true. If so I would rename the existing term and add a definition.

**Comment by pnrobinson** [comment]
I think these are synoynms. I am replacing the primary label and adding the definition you suggested. The term will be HP_0006931 and should be online with our next release in October.

**Comment by DBHI-BiG** [comment]
Fantastic and thank you￼
